contracts/src/messaging/messenger/ICrossChainMessenger.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
not sure I like the "messenger" name. 
I doesn't really give me any information about what this stuff is.

This is a wrapper over the message bus .

can we find a better name?

Reply:
I will think about it nothing on the top of my head. I picked it since it's the naming used in other layer 2/cross chains (not all of them, but) so it is close to standard naming. And the contract itself is actually a messenger; it delivers messages and that's all there is to it.